public/scripts/acquisition.js

```diff
@@ -27,31 +27,25 @@ function init() {
 
 }
 
+//on keyup, this function is called, but why? what does it do?
 function enterSession(key) {
-		
-
     document.getElementById("sessionInput").style.display = "none";
     document.getElementById("sessionLabel").style.display = "none";
     document.getElementsByClassName("resetButton")[0].style.display = "block";
 
-		
-
-
     let exists = 0;
 		
-	
+	//This call checks to see if the "key" exists inside the database
```

Comment:
yes, this call checks whether the realtime database entry with the name of entered session ID exists